xml/public-cloud-setup.xml

```diff
@@ -0,0 +1,565 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="public.cloud"
+xmlns="http://docbook.org/ns/docbook"
+xmlns:xi="http://www.w3.org/2001/XInclude"
+xmlns:xlink="http://www.w3.org/1999/xlink">
+  <info>
+    <title>Public Cloud Setup</title>
+    <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+      <dm:bugtracker></dm:bugtracker>
+      <dm:translation>yes</dm:translation>
+    </dm:docmanager>
+  </info>
+  <para>For detailed information about the product please refer to
+  the https://www.suse.com/documentation/suse-caasp-2
+  &amp;productname; documentation.</para>
+  <para>The &amp;productname; images published by SUSE in selected
+  Public Cloud environments are provided as Bring Your Own
+  Subscription (BYOS) images. &amp;productname; instances need to
+  be registered with the SUSE Customer Center (SCC) in order to
+  receive bugfix and security updates. Images labeled with the
+  <emphasis role="bold">cluster</emphasis> designation in the name
+  are not intended to be started directly; they are deployed by the
+  Administrative node. Administrative node images contain the
+  <emphasis role="bold">admin</emphasis> designation in the image
+  name.</para>
+  <sect1 xml:id="instance-requirements">
+    <title>Instance Requirements</title>
+    <para>In Amazon EC2, the adminstrative instance must be
+    launched with an IAM role that allows full access to the EC2
+    API. In Microsoft Azure, all security credentials will be
+    collected during setup. In Google Compute Engine, the instance
+    must be launched with an IAM role including
+    <emphasis role="literal">Compute Admin</emphasis> and
+    <emphasis role="literal">Service Account
+    Actor</emphasis> scopes.</para>
+    <para>Select an instance size for the Administrative node that
+    meets the system requirements as documented in the
+    &amp;productname; documentation.
+    <variablelist>
+      <varlistentry>
+        <term>Memory</term>
+        <listitem>
+          <para>Minimal main memory: 8GB</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Volume Size</term>
+        <listitem>
+          <para>Root volume size should be at least 40GB and is
+          dependent on the size and number of different containers
+          running in the cluster. The default root volume size of
+          the images is smaller than 40GB in some Public Cloud
+          frameworks; you must resize the root volume to meet your
```

Comment:
Just to clarify - are you referring to needing to run a resizefs on an existing snapshot?

Reply:
No, this works via the framework command line tools or the web UI, but clarified. Thanks